examples/tictactoe/ts/src/index.ts

```diff
@@ -10,41 +10,26 @@
  * Run with node:     `$ node build/src/index.js`.
  */
 
-import {
-  Field,
-  PrivateKey,
-  Bool,
-  Mina,
-  Party,
-  shutdown,
-  Signature,
-  isReady,
-} from 'snarkyjs';
+import { Field, PrivateKey, Mina, shutdown, isReady } from 'snarkyjs';
+import { createLocalBlockchain, makeMove, deploy } from './tictactoe-lib.js';
 import { TicTacToe, Board } from './tictactoe.js';
 
-async function main() {
+async function playTicTacToe() {
   await isReady;
 
-  const Local = Mina.LocalBlockchain();
-  Mina.setActiveInstance(Local);
-
-  const player1 = Local.testAccounts[0].privateKey;
-  const player2 = Local.testAccounts[1].privateKey;
+  const accounts = createLocalBlockchain();
+  const player1 = accounts[0];
+  const player2 = accounts[1];
```

Comment:
nit: you could use array destructuring to condense this a bit.  `const [player1, player2] = createLocalBlockchain()`